api/src/db/seeds.rb

```diff
@@ -7,8 +7,32 @@
 #   Character.create(name: "Luke", movie: movies.first)
 
 # Create a user
-user = User.create(
-    name: "Test",
-    email: "test@gmail.com",
-    password: "password",
+User.create!(
+  [
+    {
+      name: "Admin",
+      email: "admin@gmail.com",
+      password: "password",
+      admin: true
```

Comment:
```suggestion
      isAdmin: true
```
にしてほしい。基本的にboolean変数とかはisとかhasとかつけて、関数とかはgetとかfetchとか先に動詞を書いてその後に何をするかで変数名は決めてほしいな